lib/src/components/NxToast/NxToast.scss

```diff
@@ -39,12 +39,12 @@
   right: 0;
   opacity: 1;
   margin-top: var(--nx-spacing-2x);
-  animation: animationIn 0.4s linear;
+  animation: animationIn 300ms ease-out forwards;
   overflow: hidden;
   pointer-events: auto;
 
   &.nx-toast--closing {
-    animation-name: animationOut;
+    animation: animationOut 250ms ease-in forwards;
```

Comment:
Specifying the `animation-fill-mode` (the `forwards`) doesn't seem necessary to me, I think it's working without it.

Reply:
> Specifying the `animation-fill-mode` (the `forwards`) doesn't seem necessary to me, I think it's working without it.

I removed it here: cbfa9f3d3854e60cc99e25d18e159c831eb5e65e